README.md

```diff
@@ -68,9 +68,30 @@ If `cmake` cannot find your `uWS` library, try providing it manually to `cmake`
 |  Option          | Description    |
 | :--------------  | ----------:    |
 | `BUILD_TESTS`    | Build tests    |
-| `BUILD_EXAMPLES` | Build examples |    |
+| `BUILD_EXAMPLES` | Build examples |
 | `BUILD_DOCS`     | Build docs     |
 
+### Run examples using Docker
+
+#### Build the image
+
+In project directory,
+
+`docker build -t <IMAGE-NAME> .`
+
+#### Run the built image
+
+`docker run -it -e ENV1_NAME=ENV1_VALUE <IMAGE-NAME>`
```

Comment:
Let's tweak this to 

`docker build -t cppkiteconnect-dev .`

`docker run -it -e EXAMPLE_NUMBER=1 cppkiteconnect-dev`

Makes it easier for users.